README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion: A method that calls itself. ("See recursion" - haha)
+- Recursive Case: Part of the method where the recursion happens, all the instances we need to loop/de recursion. Need at least one recursive case for it to be a recursion. There can be multiple recursive cases.
+- Base Case: Stopping case, when there are no more recursions left to make. Need at least one base case for it to be a recursion. There can be multiple base cases.
+- Activation Chain/Stack: All the times the recursion method was called.
+- Activation Record/Call: Each instance of the recursion method.
+- Infinite Recursion/Stack Overflow/Stack too deep: Method calling another method (could be itself) too many times. Computer has run out of space.
+- Tail Recursion: Skipped during lecture. From what I found online, tail recursion is special type of recursion that is faster because it doesn't need to go through the activation chain. The recursive case is the last thing called in the method. Because no further operations are called after the recursive case, it's less the computer needs to keep track of. I believe all of these example exercises are tail recursions because the recursive case is at the end of each method.
```

Comment:
Hmm, all of your answer makes sense, except the last sentence. None of these are tail recursive problems. 

Here's and example of a tail recursive problem:
```ruby
def mystery7(word)
  if word.length < 2
    return true
  elsif word[0] != word[-1]
    return false
  else
    return mystery7(word[1..-2])
  end
end
```
Note that once we hit true or false, we just return that up the chain with no further computations on the way up.